index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+    <main>
+      <div class="container">
+        <div class="name">
+          <ul class="na">
+            <li>Emma Ngufor</li>
+          </ul>
+        </div>
+  
+        <div class="name1">
+          <ul class="name2">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><img src="image/Union.png" alt="an envelope"></li>
+          </ul>
+        </div>
+      </div>
```

Comment:
Kindly wrap the navigation menu in the `<header>` tag and the rest of the page in the main tag for semantic reasons 😃

- _Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info._

Also, please update your header design to look like the Figma Design. Replace the Navigation bar with a hamburger menu. Here is a comparison:

- Your design
![image](https://user-images.githubusercontent.com/73714615/185572995-907ccae4-ea5a-4ad9-b303-2425c899fa8a.png)
- Figma design
![image](https://user-images.githubusercontent.com/73714615/185573044-955391ea-d9b3-494a-87e6-464f9dbe50c5.png)